src/c.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2016-2019 Brian Smith.
+//
+// Permission to use, copy, modify, and/or distribute this software for any
+// purpose with or without fee is hereby granted, provided that the above
+// copyright notice and this permission notice appear in all copies.
+//
+// THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHORS DISCLAIM ALL WARRANTIES
+// WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+// MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY
+// SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+// WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION
+// OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
+// CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+//! C types.
+//!
+//! The libc crate provide the C types for most, but not all, targets that
+//! *ring* supports.
+
+use libc;
+
+pub(crate) type size_t = libc::size_t;
+pub(crate) type int = libc::c_int;
```

Comment:
Rust has a `target-c-int-width` attribute that we could use to define `c::int` instead of relying on `libc`. However, this is best addressed in a future PR.

Reply:
> Rust has a `target-c-int-width` attribute that we could use to define `c::int` instead of relying on `libc`. However, this is best addressed in a future PR.

That would be awesome to use if it works! Can you give an example? I tried `#[cfg(target_c_int_width = "32")] type int = i32;` but `target_c_int_width` doesn't seem to be exposed for `#[cfg(...)]` to use.